sdks/java/io/pulsar/src/main/java/org/apache/beam/sdk/io/pulsar/PulsarWriter.java

```diff
@@ -0,0 +1,41 @@
+package org.apache.beam.sdk.io.pulsar;
```

Comment:
Can we rename this to be consistent with the ReadDoFn?

Reply:
Will do